app/controllers/movement/MovementSummaryController.scala

```diff
@@ -41,6 +44,7 @@ class MovementSummaryController @Inject()(
   authenticate: AuthAction,
   errorHandler: ErrorHandler,
   customsCacheService: CustomsCacheService,
+  customsDeclareExportsConnector: CustomsDeclareExportsConnector,
```

Comment:
My propose is to create `CustomsDeclareExportsMovementConnector` and not use connector for backend, because we're not connecting through the backend to movements, but directly to movements.

Reply:
have changed this